<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSysOptions</class>
 <widget class="QTabWidget" name="FrmSysOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opciones del Sistema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#topFrame, #topFrame_2  {
background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(37, 77, 136, 255), stop:1 rgba(58, 116, 177, 255));
padding: 0;
}

#topFrame QPushButton, #topFrame_2  QPushButton {
border: none;
padding: 3px 5px;
margin: 0;
background-color: none;
color:white;
}

#topFrame QPushButton:hover, #topFrame_2 QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, 
radius: 1.35, stop: 0 #fff, stop:  1 #bbb);
color : rgba(58, 116, 177, 255);
}

#topFrame QPushButton:pressed, #topFrame_2 QPushButton:pressed {
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, 
radius: 1.35, stop: 0 #fff, stop:  1 #ddd);
}</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabTablas">
   <attribute name="title">
    <string>Tablas del Sistema</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Bancos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Categorias</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marcas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tallas</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="topFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#topFrame {
background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(37, 77, 136, 255), stop:1 rgba(58, 116, 177, 255));
padding: 0;
}

#topFrame QPushButton {
border: none;
padding: 3px 5px;
margin: 0;
background-color: none;
color:white;
}

#topFrame QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, 
radius: 1.35, stop: 0 #fff, stop:  1 #bbb);
color : rgba(58, 116, 177, 255);
}

#topFrame QPushButton:pressed {
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, 
radius: 1.35, stop: 0 #fff, stop:  1 #ddd);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="btnNuevo">
           <property name="text">
            <string>&amp;Nuevo</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/images/standardtoolbar/document-new.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGuardar">
           <property name="text">
            <string>&amp;Guardar</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/images/standardtoolbar/document-save.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="bancos">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0" rowspan="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Nombre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtBancoNombre"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Bancos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QTableView" name="tbwBancos">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="categorias">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Categorias&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Nombre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtCategoriaNombre"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QTableView" name="tbwCategorias">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="marcas">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0" rowspan="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Nombre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtMarcaNombre"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Marcas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QTableView" name="tbwMarcas">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tallas">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Tallas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Nombre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtTallaNombre"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QTableView" name="tbwTallas">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabConfiguracion">
   <attribute name="title">
    <string>Configuracion del Sistema</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
